File: 8.8/example1.py
count = 0 # Swtting the value of count to 0
continueInput = "Yes" # Setting the value of continueInput to Yes
while continueInput == "Yes": # Running a loop while continueInput is Yes
    continueInput = input("Do you want to continue") # Setting the value of continueInput to the value inputed
    count = count + 1 # Adding 1 to the value of count
print("You continued ", str(count - 1), " times") # Printing

File: java/8.8/example1.java
import java.util.Scanner;

public class example1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Integer count = 0;
        String continueInput = "Yes";
        while(continueInput.equals("Yes")) {
            System.out.println("Do you want to continue");
            continueInput = scanner.nextLine();
            count++;
        }
    }
}
